haiku-apps/timetracker/timetracker-0.2.recipe

```diff
@@ -7,12 +7,15 @@ HOMEPAGE="https://github.com/HaikuArchives/TimeTracker"
 COPYRIGHT="200? Michael Wulff Nielsen
 	2016 Humdinger"
 LICENSE="GNU GPL v1"
-REVISION="1"
+REVISION="2"
```

Comment:
Thanks @nysnatuss for the update!

Hmm, may I ask you a last tiny change? (Please wait before you change anything, because the change I'm suggesting below might not be wanted.)
Could you please revert timetracker's `REVISION` bump?
It's not needed because all the following conditions are met:
* the recipe:did not change for x86_gcc2 (as we found that we do not want the patch on that arch, otherwise it would break the app here);
* on all other architectures, i.e. x86 and x86_64, the recipe did change... but it was previously marked as broken, so this means no automatic attempt (on our builders) was ever made to build this app on these architectures (with the current version and `REVISION`).

**_Question to the maintainers of the build servers:_** do you need a `REVISION` bump on timetracker to trigger a build of the app on our builders?

@nysnatuss, in case the answer to the previous question is yes, everything is OK and I will be able to merge your PR as is.
If the answer is no, I'll wait until you make the change, but there is no urgency. If, for any reason, you cannot do the change, that's OK, as I can update your remote branch myself and then merge the PR :)
Thanks!

Reply:
> do you need a REVISION bump on timetracker to trigger a build of the app on our builders?

No, the change to the recipe is enough. Note that it will still be scheduled in a buildrun on the other archs,  but will be skipped there due to the missing revision bump. Thid is fine if the package really didn't change for the archs where it was already built. If it did change, even just the description or summary or more importantly any of the provides/requires, the package must however be rebuilt (and therefore the revision bumped).

Reply:
OK, thanks @mmlr for your answer and detailed explanation.
This is cool! Now I know (and understand) exactly how the builders work :)

@nysnatuss, could you set `REVISION="1"` for timetracker ? Thanks!